vchernetski/jquery.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<HTML>
+   <HEAD>
+   		<META charset="utf-8">
+   		<TITLE>jquery</TITLE> 
+   		<script type="text/javascript" src="https://code.jquery.com/jquery-3.3.1.js"></script>
+   		<script type="text/javascript">
+   			$(document).ready(function(){
+   				var a= null;
+   				var b= null;
+   				$('#p1').keyup(function(){
```

Comment:
you can use #p1, #p2 selector, or put your inputs into form tag and use $('form').keyup(function() to make sort your code